internal/operators/manager.go

```diff
@@ -401,6 +403,29 @@ func (mgr *Manager) GetSupportedOperatorsByType(operatorType models.OperatorType
 	return operators
 }
 
+func (mgr *Manager) EnsureOperatorArchCapability(cluster *common.Cluster, operators []*models.MonitoredOperator) error {
+	var monitoredOperators []string
+	if cluster.CPUArchitecture == common.X86CPUArchitecture {
+		return nil
+	}
+
+	for _, monitoredOperator := range operators {
+		monitoredOperators = append(monitoredOperators, monitoredOperator.Name)
+	}
+
+	for operatorName, operator := range mgr.olmOperators {
+		if !funk.ContainsString(monitoredOperators, operatorName) {
+			continue
+		}
+
+		if !operator.HasArmSupport() {
+			return errors.Errorf("Currently, you cannot install %s operator along side %s architecture.", operator.GetName(), cluster.CPUArchitecture)
```

Comment:
@nirfarkas in the UI we have the following message: "OpenShift Virtualization is not available when ARM CPU architecture is selected" see: https://github.com/openshift-assisted/assisted-ui-lib/blob/4829bc07b9a69be382113ba60d407544aa73931e/src/ocm/components/featureSupportLevels/featureStateUtils.ts#L109 
Can you review the BE message?

Reply:
@gamli75 After discussing this with @eliorerz we've decided to align the bad request message with the validations message so both will be:
"<operator name> is not available when <CPU arcitecture> architecture is selected"